graph/auxiliary/subgraph.yaml.mustache

```diff
@@ -0,0 +1,49 @@
+specVersion: 0.0.1
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: Anchor
+    network: mainnet
+    source:
+      address: "{{anchorAddress}}"
+      abi: Anchor
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.2
+      language: wasm/assemblyscript
+      entities:
+        - StateRootAvailable
+      abis:
+        - name: Anchor
+          file: ./abis/Anchor.json
+      eventHandlers:
+        - event: StateRootAvailable(uint256,bytes32)
+          handler: handleStateRootAvailable
+      file: ./src/mapping.ts
```

Comment:
Sharing Thoughts: Having a separate mapping file for each data source would ease reviewing those files in case of need.

Reply:
Make sense, Done